CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### Release [1.3.3], 2023-01-18
+#### Bug Fix
+- Client connections would incorrectly reuse a previous session_id when initializing, causing session locked errors.  This has been fixed.  This
```

Comment:
Should we explain multiple threads can be used from now on?

Reply:
yes, done.